exporter/f5cloudexporter/factory.go

```diff
@@ -122,3 +131,7 @@ func getTokenSourceFromConfig(config *Config) (oauth2.TokenSource, error) {
 
 	return ts, nil
 }
+
+func setVersionInUserAgent(cfg *Config, version string) {
```

Comment:
How about `fillUserAgent` or `resolveUserAgentVars` since we can expect adding more variables eventually?

Reply:
Great recommendation, @anuraaga! I'll update now.